examples/tutorial/src/client.ts

```diff
@@ -0,0 +1,13 @@
+import { EnvOptions, VocdoniSDKClient } from '@vocdoni/sdk'
+import { Wallet } from '@ethersproject/wallet'
+
+export const getDefaultClient = () => {
+  const wallet = Wallet.createRandom()
+  const client = new VocdoniSDKClient({
+    env: EnvOptions.STG,
+    api_url: 'https://api-stg.vocdoni.net/v2',
```

Comment:
If `env` is defined, there is no need to give the API URL. The `api_url` is intended for custom deploys. Said that, all the initialization parameters from the SDK should be reviewed and arranged in a better way: https://github.com/vocdoni/vocdoni-sdk/issues/243